### DISCUSSION TITLE
Best settings for long lasting stream?

### DISCUSSION THREAD

**Discussion by Toei79** [GerryDazoo/Slinger#443] [General]
ok so far i try 2 idk what change now

i was using phone connection. so it was remotely with port forward in my local network.

using this one i get 30 minutes of fluid stream , i let play three times before need restart the stream

**Resolution=5
FrameRate=30
VideoBandwidth=2048
VideoSmoothness=59
IframeRate=5
VideoSource=2
;RemoteLock=yes
;StartChannel=155
;Remote=solo_remote.txt**

then i try this one i get 15 minutes of fluid stream. i let play twice

**Resolution=5
FrameRate=30
VideoBandwidth=2048
VideoSmoothness=56
IframeRate=4
VideoSource=2
;RemoteLock=yes
;StartChannel=155
;Remote=solo_remote.txt**

in both cases i try remote connection not local. 
im using 500 model.
`

**Comment by sp222222** [comment]
try:
16
1984
59
4

**Reply by Toei79** [reply]
Thsnks. I'm going to try andThanks.

16 its framerate?

**Reply by flipper2222222** [reply]
&sp222222

I ask the same at Toei79 , are you saying 16 for the frame rate?

**Comment by mdehrlich** [comment]
If you can  tolerate the image, drop the framerate to 25 or 20.  Bandwidth does not seem to do anything.  Also I believe Iframe rate of 5 is better than 4.  It means that every 5 seconds, an iframe is sent as a complete image.  Doing this every 4 seconds means a higher bandwidth is required.

**Reply by GerryDazoo** [reply]
I think the I-Frame rate  is seconds not frames.

**Reply by mdehrlich** [reply]
ok, corrected.

**Comment by flipper2222222** [comment]
would anyone else like to chime in with what they use as these settings , I use these for my 500 usually OK, but I would try other settings if others report back with suggestions. 
FrameRate=60
VideoBandwidth=2048
VideoSmoothness=59
IframeRate=4

**Reply by blasco403** [reply]
I use the following to stream remote with a 350 and 500 and it works great. 

FrameRate=30
VideoBandwidth=1984
VideoSmoothness=59
IframeRate=4
